spec/bolt/module_installer/puppetfile/module_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'bolt/module_installer/puppetfile/module'
+
+describe Bolt::ModuleInstaller::Puppetfile::Module do
+  let(:name) { 'puppetlabs/yaml' }
+  let(:mod)  { described_class.new(name) }
+
+  context '#initialize' do
+    it 'normalizes the full name' do
```

Comment:
I think this should verify that `puppetlabs-yaml` gets converted to `puppetlabs/yaml`